### PR TITLE
flac: add page

### PR DESCRIPTION
**Pull request by AberDerBart**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/flac.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# flac
+
+> Encodes, decodes and tests flac files.
+
+
+- Encode a wav file to flac, writing to /path/to/file.flac:
+
+`flac {{/path/to/file.wav}}`
+
+- Decode a flac file to wav, specifying the output file:
```
Comment (agnivade) [inline pages/common/flac.md]: We should specify the `--output-name` flag with the encode first and then move to decode.

File: pages/common/flac.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`flac -d --output-name {{/path/to/output.wav}} {{/path/to/input.flac}}`
+
+- Test a flac file:
```
Comment (agnivade) [inline pages/common/flac.md]: What does it test for ?

Comment (AberDerBart) [reply pages/common/flac.md]: for correct encoding - if there are any remarkable errors in the file, they are reported. I added a short remark in the example.

**Comment by AberDerBart** [comment]
Looking good, thank you!

**Pull request merged**